options/posix/include/sys/mman.h

```diff
@@ -1,12 +1,14 @@
 #ifndef _SYS_MMAN_H
 #define _SYS_MMAN_H
 
+#include <abi-bits/mode_t.h>
 #include <abi-bits/vm-flags.h>
 #include <bits/off_t.h>
 #include <bits/size_t.h>
 
 #define MAP_FAILED ((void *)(-1))
 
+#define MAP_FILE 0 // Linux uses 0 too
```

Comment:
This is a ABI-specific constant, thus a comment about Linux does not make sense -- on Linux we *have* to use 0. POSIX-compliant systems will define this as 0 (since MAP_FILE is the default behavior according to POSIX). Drop the comment.